jira/client.py

```diff
@@ -991,18 +986,17 @@ def file_stream() -> MultipartEncoder:
                     fields={"file": (fname, attachment, "application/octet-stream")}
```

Comment:
`attachment` here is IO stream, I wonder if this needs to `.seek(0)` so that on retries it actually works...

Reply:
yep